CHANGELOG.md

```diff
@@ -4,13 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [5.8.0]
+### Added
+- Add section to trigger wallet_watchAsset ([#222](git+https://github.com/MetaMask/test-dapp/pull/222))
+
+### Changed
+- Update minimum Node.js version to v16 ([#225](git+https://github.com/MetaMask/test-dapp/pull/225))
```

Comment:
We typically treat changes to the minimum supported Node.js version as breaking; could you update this to use v6.0.0 as the release version?

And we can indicate the breaking change like this:

```suggestion
- **BREAKING:** Update minimum Node.js version to v16 ([#225](git+https://github.com/MetaMask/test-dapp/pull/225))
```

Reply:
Done.